contracts/core/OriginCore.sol

```diff
@@ -41,6 +42,17 @@ contract OriginCore is OriginCoreInterface, OriginCoreConfig {
         bytes32 indexed kernelHash,
         bytes32 transitionHash
     );
+    /** Emitted whenever a vote is verified for proposed meta-block. */
+    event VoteVerified(
+        bytes32 indexed kernelHash,
+        bytes32 transitionHash,
+        address validator,
+        bytes32 voteHash,
+        uint8 v,
+        bytes32 r,
+        bytes32 s,
+        uint256 totalWeight
```

Comment:
it would probably be interesting to know the verified weight *and* the required weight.